src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -140,4 +140,49 @@
             }
         }
     }
+}
+
+#ctZone {        
+    display: grid;
+    width: calc(100% - 40px);
+    height: 600px;
+    grid-template-columns: 64px 1fr;
+    grid-template-rows: 1fr;
+
+    margin:20px;
+    background: rgb(64,64,64);
+}
+
+#toolbarzone {
```

Comment:
FYI - long term we are going to use the toolbar integrated into the editor page vs. the canvas tools.

Reply:
This was placeholder until integration with toolbar

Reply:
This was placeholder until integration with toolbar

Reply:
For long term:
1) can we ensure that we have something like toolbar sets, like expose tools for box selection/manipulation vs expose tools for free-form/polygon selection, etc. I think it shoulbe some how related to the current tagging job.
2) I'm thinking on some options for something like plugins for canvastools that could allow adding new custom features (e.g., new selector types like cubes). Do you think we can have some APIs in future to extend the toolbar and shortcuts registration as well?